src/pages/ExerciseDetails/SideGallery/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from './side_gallery.module.css';
+
+export const SideGallery = () => {
```

Comment:
We can just use something like https://mui.com/components/image-list/#main-content (or probably better to just use MUIs grid system, exercises won't have a long list of images, probably just a handful at most)

Reply:
OK, will change that